src/gsi/read_satwnd.f90

```diff
@@ -1089,7 +1180,9 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
                 if(itype==251 )  then;  c_prvstg='GOESR' ; c_sprvstg='VIS'  ; endif
               endif
 ! Extra block for GOES-R winds: End
-
+           else ! wind is not recognised and itype is not assigned
+              write(6,*) 'READ_SATWND:SUBSET,itype, satID: ',trim(subset),itype,hdrdat(1)
```

Comment:
Is reaching this write statement a "normal' thing or do we want the write statement to be worded more strongly (e.g, include "***WARNING***"?

Reply:
It is acceptable for a satwnd bufr file to contains winds that are not currently read if/when there are plans to be evaluated and possibly used. Such decision lead to keeping INSAT winds in the production satdumps. However, I also see the value of including "WARNING" -  it is a clear "inventory" of what is in the satwnd file, and what is used from him. I will add that.